### ISSUE TITLE
 3.13.0 RC1 crashes, when I choose an E2EE-encrypted folder for auto upload

### ISSUE THREAD

**Issue by spackmat** [nextcloud/android#6518]
I tried to test auto upload with an E2EE encrypted folder in 3.13.0 RC1, but the app crashes when I select an encrypted folder. When I choose an unencrypted folder, it works as expected. I get the error log below.

Maybe related to #6417, maybe not.

### Steps to reproduce
1. Have a working E2EE folder in 3.13.0 RC1
2. Enable auto upload for a folder and select that encrypted folder
3. Oberve the app crashing

### Expected behaviour
- Choosing an encrypted folder for auto upload should work as before (old E2EE preview) or as with an unencrypted folder.

### Actual behaviour
- The app crashes every time I choose an E2EE encrypted folder as target for auto upload.

### Can you reproduce this problem on https://try.nextcloud.com?
- Didn't test as I don't expect that to have the fresh E2EE beta stuff installed.

### Environment data
Android version: 10

Device model: Xiaomi Mi Mix 3

Stock or customized system: Stock

Nextcloud app version: 3.13.0 RC1

Nextcloud server version: 19.0.1

Reverse proxy: No

#### Nextcloud log (data/nextcloud.log)
Crash report log:
```
************ CAUSE OF ERROR ************

java.lang.NullPointerException: Attempt to invoke virtual method 'void com.google.android.material.floatingactionbutton.FloatingActionButton.setOnClickListener(android.view.View$OnClickListener)' on a null object reference
	at com.owncloud.android.ui.fragment.OCFileListFragment.registerFabListener(OCFileListFragment.java:422)
	at com.owncloud.android.ui.fragment.OCFileListFragment.updateLayout(OCFileListFragment.java:1302)
	at com.owncloud.android.ui.fragment.OCFileListFragment.listDirectory(OCFileListFragment.java:1272)
	at com.owncloud.android.ui.fragment.OCFileListFragment.listDirectory(OCFileListFragment.java:1214)
	at com.owncloud.android.ui.fragment.OCFileListFragment.onItemClicked(OCFileListFragment.java:951)
	at com.owncloud.android.ui.adapter.OCFileListAdapter.lambda$onBindViewHolder$1$OCFileListAdapter(OCFileListAdapter.java:379)
	at com.owncloud.android.ui.adapter.-$$Lambda$OCFileListAdapter$pouWgEOOHT1aDFHaHM0Ifm3xWtY.onClick(Unknown Source:4)
	at android.view.View.performClick(View.java:7161)
	at android.view.View.performClickInternal(View.java:7138)
	at android.view.View.access$3500(View.java:811)
	at android.view.View$PerformClick.run(View.java:27419)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7520)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30130051
Build flavor: generic
```

**Comment by spackmat** [comment]
The new 3.13.0 RC2 (30130052) still crashes when I select an E2EE-encrypted folder.

**Comment by tobiasKaminsky** [comment]
Fixed via #6623

**Comment by spackmat** [comment]
With the new 3.13.0 RC3 (30130053) I could successfully set the camera upload to an E2EE-encrypted folder. Thank you!
